integration-tests/src/test/java/org/zowe/apiml/integration/authentication/pat/PATWithAllSchemesTest.java

```diff
@@ -0,0 +1,259 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package org.zowe.apiml.integration.authentication.pat;
+
+import io.restassured.response.Response;
+import org.apache.http.HttpHeaders;
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import org.zowe.apiml.constants.ApimlConstants;
+import org.zowe.apiml.util.SecurityUtils;
+import org.zowe.apiml.util.categories.InfinispanStorageTest;
+import org.zowe.apiml.util.config.ItSslConfigFactory;
+import org.zowe.apiml.util.config.SslContext;
+import org.zowe.apiml.util.http.HttpRequestUtils;
+
+import java.net.URI;
+import java.util.Base64;
+import java.util.HashSet;
+import java.util.Set;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasKey;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.zowe.apiml.util.SecurityUtils.*;
+import static org.zowe.apiml.util.requests.Endpoints.*;
+
+public class PATWithAllSchemesTest {
```

Comment:
The three nested classes are very similar. What about making them parametrized (or extended). One different is the input credentials.